app/assets/v2/css/base.css

```diff
@@ -583,7 +583,6 @@ input[type=text].loading {
 
 .alert .message {
   align-items: center;
-  display: flex;
 }
```

Comment:
Instead of this I suggest changing the `_alert` function in shared.js to something like:

```javascript
var _alert = function(msg, _class) {
  if (typeof msg == 'string') {
    msg = {
      'message': msg
    };
  }
  var numAlertsAlready = $('.alert:visible').length;
  var top = numAlertsAlready * 66;

  var html = function() {
    return (
      `<div class="alert ${_class}" style="top: ${top}px">
        <div class="message">
          <div class="content">
            ${alertMessage(msg)}
          </div>
        </div>
        ${closeButton(msg)}
      </div>;`
    );
  };

  $('body').append(html);
};
```

The extra `<div class="content">` ensures that the content remains unharmed while also maintaining the flex property between the icon and the content.

cc: @thelostone-mc @mbeacom

Reply:
d5b87be617af5a82e87ecd90e33c0996cdb55c3f